source/Nevermore/RelationalStore.cs

```diff
@@ -37,7 +51,8 @@ public class RelationalStore : IRelationalStore
             JsonSerializerSettings jsonSettings,
             IRelatedDocumentStore relatedDocumentStore,
             int keyBlockSize = 20,
-            ObjectInitialisationOptions objectInitialisationOptions = ObjectInitialisationOptions.None)
+            ObjectInitialisationOptions objectInitialisationOptions = ObjectInitialisationOptions.None,
+            bool setMarsAsDefault = true)
```

Comment:
May consider rename it to `isMarsEnabledByDefault or `enableMarsByDefault` as `setMarsAsDefault` sounds like a action.

Reply:
👍